psp/Pv.py

```diff
@@ -458,6 +458,11 @@ def monitor(self, mask=pyca.DBE_VALUE | pyca.DBE_LOG | pyca.DBE_ALARM,
         count : int, optional
             Subsection of waveform record to monitor. By default,
             :attr:`.count` is used
+         
+         wait_for_init : bool, optional
```

Comment:
It looks like you also added a new argument here, not mentioned in the PR text. It seems sensible and backwards-compatible.

Small docstring formatting here, mismatcehd indentation level:
```suggestion
        wait_for_init : bool, optional
```